src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx

```diff
@@ -124,7 +124,10 @@ export const ConversationsList = ({
       }
 
       clearSearchFilter();
-      clickedFilteredConversation.current = conversation;
+
+      clickedFilteredConversation.current = document.querySelector<HTMLElement>(
+        `.conversation-list-cell[data-uie-uid="${conversation.id}"]`,
+      );
```

Comment:
I am curious what if the element is not there in the DOM.
I remember, we faced this issue that at the time of click the element was not in the DOM.